components/AudioPlayer/AudioInterface.js

```diff
@@ -0,0 +1,259 @@
+import React, { useEffect, useState, useRef } from "react";
```

Comment:
We don't need to import `React` anymore. 

```suggestion
import { useEffect, useState, useRef } from "react";
```